index.html

```diff
@@ -171,7 +124,7 @@ <h3 class="mb-0">Events & Meetings</h3>
 
       <!-- Blog -->
       <article class="row g-5">
-        <div class="col-9">
+        <div class="col-sm-12 col-md-9">
```

Comment:
Unsure why this line is marked as modified as new... it's already checked into dev with the bootstrap mods

Reply:
Yes. I am also not able to understand. What can I do now?